.github/workflows/pr-build.yml

```
@@ -1,5 +1,7 @@
 name: build and validate
 on:
+  schedule:
+    - cron: '11 11 * *3'
```

typo (also quotes not really qrequired
```suggestion
    - cron: 11 11 * * 3
```